content/events/contents.lr

```diff
@@ -8,3 +8,4 @@ body:
 
 - Tuesday, October 29th: [Code Night - Kattis Competition](./2019/kattis_comp/)
 - Sunday, March 31st: [MUNCS + Chess.NL - March Rapid](./2019/march_rapid/)
+- Friday, November 15th: [Game Night! - Xonotic LAN](./2019/xonotic_lan/)
```

Comment:
Move lan to above kattis comp, so the events are in order

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d